docs/source/user_guide/01a_filestructure.rst

```diff
@@ -0,0 +1,63 @@
+
+Recommended file structure
+==========================
+For instructor ease of use and developer maintenance, nbgrader makes a few
+assumptions about how your assignment files are organized. By default, nbgrader
+assumes that your assignments will be organized with the following directory
+structure:
+
+::
+
+    {course_directory}/{nbgrader_step}/{student_id}/{assignment_id}/{notebook_id}.ipynb
+
+course_directory
```

Comment:
Maybe this could just be a bulleted list? I don't think everything needs it's own section, and then we can have separate bullets for `student_id`, `assignment_id`, and `notebook_id`.